json-rpc/json-rpc-spec.md

```diff
@@ -633,11 +713,11 @@ A Libra account.
    </td>
   </tr>
   <tr>
-   <td>balance
+   <td>balances
    </td>
-   <td><a href="#amount---type">Amount</a>
+   <td><a href="#amount---type">Array of Amount objects </a>
```

Comment:
the <a> should still be on Amount

Reply:
U mean hyper-link should be only on Amount but the txt can be "Array of Amount objects" Or it should just point to Amount?

Reply:
the hyperlink should only contain "Amount" as that is the type

Reply:
Changed it back to Amount

Reply:
no man, i meant it should be 
```
   <td><span>Array of <a href="#amount---type">Amount</a> objects </span>
```

Reply:
My bad :-(. Fixed it!